doc/modules/linear_model.rst

```diff
@@ -136,17 +136,27 @@ Setting the regularization parameter: generalized Cross-Validation
 ------------------------------------------------------------------
 
 :class:`RidgeCV` implements ridge regression with built-in
-cross-validation of the alpha parameter.  The object works in the same way
+cross-validation of the alpha parameter. The object works in the same way
 as GridSearchCV except that it defaults to Generalized Cross-Validation
 (GCV), an efficient form of leave-one-out cross-validation::
 
     >>> from sklearn import linear_model
-    >>> reg = linear_model.RidgeCV(alphas=[0.1, 1.0, 10.0], cv=3)
-    >>> reg.fit([[0, 0], [0, 0], [1, 1]], [0, .1, 1])       # doctest: +SKIP
-    RidgeCV(alphas=[0.1, 1.0, 10.0], cv=3, fit_intercept=True, scoring=None,
-        normalize=False)
-    >>> reg.alpha_                                      # doctest: +SKIP
-    0.1
+    >>> reg = linear_model.RidgeCV(alphas=[1.0, 10.0], store_cv_values=True)
+    >>> reg.fit([[0, 0], [0, 0], [1, 1]], [0, .1, 1])       # doctest: +NORMALIZE_WHITESPACE
+    RidgeCV(alphas=array([ 1., 10.]), cv=None, fit_intercept=True, gcv_mode=None,
+            normalize=False, scoring=None, store_cv_values=True)
+    >>> reg.alpha_
+    1.0
+    >>> reg.cv_values_                                      # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+    array([[0.105...   , 0.259...],
+           [0.022...   , 0.125...],
+           [0.902...   , 0.902...]])
+
+In some cases, for example for very large datasets, this is not the most
+efficient way to set the hyperparameter, and k-fold cross-validation can be
+faster. Specifying the value of the `cv` attribute will trigger the use of
+cross-validation with `GridSearchCV`, for example `cv=10` for 10-fold
+cross-validation.
```

Comment:
Actually, on all the tests that I tried, Generalized CV with a non-trivial number of alphas (e.g. 5 or more) is always significantly faster than 5-fold cross-validation.

Do you have a case where GCV is not competitive with k-fold CV from a performance point of view?

Reply:
Actually I am not able to find such a case, sorry for the noise. I'll fix the
documentation. I had in mind that in the medium p, large n case, gcv was not the
fastest method, and the discussion of the technical report cited in the doc
suggests that may be the case, but I couldn't observe it. This may be due to the
fact that the `RidgeCV` with `cv != None` uses a `GridSearch` and thus doesn't
reuse the svd decompositions or benefit from a warm start for other alphas
AFAIK.

Reply:
Indeed warm-starting and reused SVD might be interesting but should better be implemented in a separate PR.